### PR TITLE
Added tests #18452

### PR DESCRIPTION
**Pull request by akhil-dh1**
#### Brief description of what is fixed or changed
Added tests for #18452 
#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by akhil-dh1** [comment]
@sachin-4099 can you please review

File: sympy/series/tests/test_limits.py [hunk 1]
```diff
@@ -737,6 +737,10 @@ def test_issue_18378():
 def test_issue_18442():
     assert limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-') == Limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-')
 
+def test_issue_18452():
+    assert limit(abs(log(x))**x, x, 0) == 1
+    assert limit(abs(log(x))**x, x, 0, "+") == 1
```
Comment (sachin-4099) [inline sympy/series/tests/test_limits.py:742]: By default the `dir` will be `+`, so the testcase having `dir` explicitly specified as `+` can be removed.

File: sympy/series/tests/test_limits.py [hunk 2]
```diff
@@ -737,6 +737,10 @@ def test_issue_18378():
 def test_issue_18442():
     assert limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-') == Limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-')
```
Comment (sachin-4099) [inline sympy/series/tests/test_limits.py:739]: Maintain two line spacing between the testcases.

File: sympy/series/tests/test_limits.py (hunk 1 above)
Comment (smichr) [inline sympy/series/tests/test_limits.py:742]:
```suggestion
```

File: sympy/series/tests/test_limits.py [hunk 3]
```diff
@@ -737,6 +737,10 @@ def test_issue_18378():
...
 
+def test_issue_18452():
+    assert limit(abs(log(x))**x, x, 0) == 1
+    assert limit(abs(log(x))**x, x, 0, "+") == 1
+    assert limit(abs(log(x))**x, x, 0, "-") == 1
```
Comment (smichr) [inline sympy/series/tests/test_limits.py:743]:
```suggestion
    assert limit(abs(log(x))**x, x, 0, "-") == 1

```

File: sympy/series/tests/test_limits.py [hunk 4]
```diff
@@ -737,6 +737,10 @@ def test_issue_18378():
 def test_issue_18442():
     assert limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-') == Limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-')
 
+def test_issue_18452():
```
Comment (smichr) [inline sympy/series/tests/test_limits.py:740]:
```suggestion

def test_issue_18452():
```

**Comment by smichr** [comment]
Please make the minor changes then squash all work to a single commit, e.g. after making modifications then, while still on this branch do:

```
git branch bak
git merge master
git diff master > dif
git checkout master
git branch -D added-tests-#18452
git checkout -b added-tests-#18452
git apply dif
git commit 'closes #18452'
git push -f akhil-dh1 added-tests-#18452
git branch -D bak
```

File: sympy/series/tests/test_limits.py (hunk 2 above)
Comment (akhil-dh1) [reply sympy/series/tests/test_limits.py:739]: I have made the modifications

File: sympy/series/tests/test_limits.py [hunk 5]
```diff
@@ -738,6 +738,11 @@ def test_issue_18442():
     assert limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-') == Limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-')
 
 
+def test_issue_18452():
+    assert limit(abs(log(x))**x, x, 0, "+") == 1
```
Comment (sachin-4099) [inline sympy/series/tests/test_limits.py:742]: This without `+` was required.

File: sympy/series/tests/test_limits.py (hunk 1 above)
Comment (sachin-4099) [reply sympy/series/tests/test_limits.py]: > By default the `dir` will be `+` in this case, so the testcase having `dir` explicitly specified as `+` can be removed.

File: sympy/series/tests/test_limits.py (hunk 5 above)
Comment (sachin-4099) [reply sympy/series/tests/test_limits.py:742]: > By default the `dir` will be `+` in this case, so the testcase having `dir` explicitly specified as `+` can be removed.

As I have mentioned here.

Comment (akhil-dh1) [reply sympy/series/tests/test_limits.py:742]: aren't both the same?

Comment (sachin-4099) [reply sympy/series/tests/test_limits.py:742]: Since they are same, that's why `+` is not required. That will be there by default.

Comment (akhil-dh1) [reply sympy/series/tests/test_limits.py]: I have corrected that.

**Comment by sachin-4099** [comment]
Looks good to me. Once the build gets passed, @smichr it can be merged.

**Review by smichr — APPROVED** [review]
looks good

**Comment by sachin-4099** [comment]
Looks good to me as well. Please merge this @smichr, once the build gets passed.

**Comment by sachin-4099** [comment]
Looks good to me as well. This can be merged @smichr.

**Pull request merged**